### ISSUE TITLE
Stops phpcbf from working

### ISSUE THREAD

**Issue by ainsofs** [eamodio/vscode-gitlens#810]
Im using [phpcbf](https://github.com/soderlind/vscode-phpcbf/issues/33) for drupal development using the drupal/coder composer library. When I turn on  [git lens](https://github.com/eamodio/vscode-gitlens) my code is not formatted correctly (using Format Document option of vscode). When I disable git lens it works again as expected. I dont know if this is a compatibility issue with here or phpcbf so posting this issue on both issue queues.

- GitLens Version: 9.9.1
  
- VSCode Version: 1.36.1
- OS Version: Windows 10 Pro 64 bit

![image](https://user-images.githubusercontent.com/11662399/62253188-919f5600-b451-11e9-8590-6e0790d23dd4.png)

Steps to Reproduce:

1. Install and setup phpcbf (as below)
1. Install git lens
1. Format document (does not work)
1. Uninstall git lens
1. Format document (works)

**My vscode settings**

```
  "phpcbf.enable": true,
  "phpcbf.executablePath": "./vendor/squizlabs/php_codesniffer/bin/phpcbf.bat",
  "phpcbf.documentFormattingProvider": true,
  "phpcbf.onsave": false,
  "phpcbf.standard": "C:\\Users\\ainsofs\\AppData\\Roaming\\Composer\\vendor\\drupal\\coder\\coder_sniffer\\Drupal",
  "[php]": {
    "editor.defaultFormatter": "persoderlind.vscode-phpcbf"
  },
```

**Comment by eamodio** [comment]
I'm unsure how GitLens could be conflicting here -- although there are certain conflicts with the python extension around some language servers not working properly with concurrent api calls. Do you have a repo or something setup that demonstrates this problem?

**Comment by eamodio** [comment]
Also can you please enable output channel logging by setting `"gitlens.outputLevel": "debug"` in your `settings.json`. This will enable logging to the *GitLens* channel in the *Output* pane. Once enabled, please reproduce the issue (if possible) and attach the log lines from the *GitLens* channel.

**Comment by ainsof** [comment]
Seems to have resolved itself in Gitlens 9.9.3
